bindings/cs/rl.net.native/rl.net.live_model.cc

```diff
@@ -65,6 +65,11 @@ API int LiveModelInit(livemodel_context_t* context, reinforcement_learning::api_
 
 API int LiveModelChooseRank(livemodel_context_t* context, const char * event_id, const char * context_json, reinforcement_learning::ranking_response* resp, reinforcement_learning::api_status* status)
 {
+	if (event_id == nullptr)
```

Comment:
This check looks unnecessary

Reply:
Ah, good catch. Under the check, it was chaining to the wrong method in the nullptr case. It is supposed to go to the auto-generate EventId code.